zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -187,7 +187,10 @@ public void addCloneParagraph(Paragraph srcParagraph) {
       String resultJson = gson.toJson(srcParagraph.getReturn());
       InterpreterResult result = gson.fromJson(resultJson, InterpreterResult.class);
       newParagraph.setReturn(result, null);
-    } catch (Exception e) { /* ignore */ }
+    } catch (Exception e) {
+      // 'result' part of Note consists of exception, instead of actual interpreter results
+      logger.info("Paragraph " + srcParagraph.getId() + " has a result with exception.");
```

Comment:
The exception and message is quite clear so I don't think we do not need trace, then I'm adding the error message!